README.md

```diff
@@ -1,26 +1,44 @@
 # geodev-hackerlabs
 
-> A place to learn how to build geo apps with the ArcGIS Platform.
+A place to learn how to build geo apps with ArcGIS.
 
-![Steps](./geodev-hackerlabs-apps.png)
+## Welcome
+
+Welcome to the Geodev Hackerlabs. This respository contains short labs that step you through the process of developing applications with [ArcGIS](http://www.arcgis.com). This process focuses mostly on using online accounts and services and does not require the installation or use of desktop software for most labs.
+
+The process generally involves three steps and the labs are broken down into the following sections:
+
+1. Data - prepare data for your apps with [ArcGIS Online](http://www.arcgis.com)
+2. Design - style layers and maps for your apps [ArcGIS Online Map and Scene Viewer](http://www.arcgis.com)
+3. Develop - consume layers and maps in your apps [ArcGIS Templates, Builders and APIs](http://developers.arcgis.com)
+
+![Process](./arcgis-process.png)
 
 ## Getting Started
 
-Sign up for a free [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/) to complete the labs. The subscription allows you to download software, register apps, import data, create new services, store maps and other content, perform spatial analysis and much, much more.
+Here's how to get started:
 
-> Note: ArcGIS Organizational accounts can also be used but a [free public account](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom) cannot be used.
+1. **Sign up for a free [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/)** - With the subscription you candownload software, register apps, import data, create new services, store maps and other content, perform spatial analysis and more. If you have an ArcGIS Organizational account, you can use that as well, but please note that [Free public accounts](https://geonet.esri.com/groups/geodev/blog/2014/10/15/help-i-cant-get-into-developersarcgiscom) cannot be used.
+
+2. **Start the labs** - The labs are divided into three sections that you can complete at your own pace. You can follow them in order or just choose the ones you want to complete. You can use the data we provide or use your own. If you want to code, dig into APIs and SDKs from Esri and compatible open source libraries. If not, you can still build highly customized production-ready apps using Esri's templates and builders. You might also find it helpful to follow along with the [Hackerlab Slides](http://slides.com/alaframboise/geodev-hackerlabs).  
```

Comment:
Oh, and I would think it's OK to edit them in slides.com but just host them here.